### PR TITLE
Use e2eskipper package in test/e2e/common/

### PR DESCRIPTION
**Pull request by YuikoTakada**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR use e2eskipper package instead of using func Skip* in kubernetes/test/e2e/common/

**Which issue(s) this PR fixes**:

Ref: #87047

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/common/secrets.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	"encoding/base64"
+
```
Comment (tanjunchen) [inline test/e2e/common/secrets.go]: /lgtm
one nit:
newline?

**Comment by oomichi** [comment]
We need to run `./hack/update-gofmt.sh` and apply the diff into this PR for passing pull-kubernetes-verify job.

**Comment by YuikoTakada** [comment]
@oomichi thanks. done

File: test/e2e/common/secrets.go (hunk 1 above)
Comment (YuikoTakada) [reply test/e2e/common/secrets.go]: thanks, done.

**Comment by YuikoTakada** [comment]
sorry, I closed PR by mistake with `git push -d origin move_skip_e2e_common` . what should I do?

**Pull request merged**